creator-node/src/tracer.ts

```diff
@@ -133,7 +138,7 @@ export const instrumentTracing = <TFunction extends (...args: any[]) => any>({
 
           // TODO add skip parameter to instrument testing function to NOT log certain args
           // tracing.setSpanAttribute('args', JSON.stringify(args))
-          const result = fn.apply(this, args)
+          const result = fn.apply(objectContext, args)
```

Comment:
feels like this could go wrong pretty easily?

Reply:
How so - it's definitely a little sus but there doesn't seem to be too many moving parts

Reply:
I actually think having an explicit this which can be passed in is good. It reduces likelihood of scoping issues because we reference the original this vs assuming it's the same in this context